scripts/apps/ingest/directives/IngestUserDashboard.ts

```diff
@@ -27,9 +27,17 @@ export function IngestUserDashboard(api, userList, privileges, moment) {
                     },
                 };
 
-                api.ingest.query(criteria).then((result) => {
-                    scope.ingested_count = result._meta.total;
-                });
+                if (scope.item.content_types.includes('event') || scope.item.content_types.includes('planning')) {
+                    const param = scope.item.content_types.includes('event') ? 'events' : 'planning';
+
+                    api.query(param, criteria).then((result) => {
+                        scope.ingested_count = result._meta.total;
+                    });
+                } else {
+                    api.ingest.query(criteria).then((result) => {
```

Comment:
you could use `api.query('ingest', ...)` here and avoid some duplication

Reply:
This whole change can be reduced down by assigning a value `resource`, using that as first argument to `api.query` and setting the value to either `events`, `planning` or `ingest` based on the `content_types` included in the ingest config